### PR TITLE
UI Refresh - Small changes

### PR DESCRIPTION
**Pull request by koredefashokun**
## Description
This PR handles minor color/theme/mobile changes, to make the UI consistent with the designs.

## Related issue
#2221 

## Motivation and Context
UI Refresh

## How Has This Been Tested?
N/A

### CODE REVIEW

**Review by avclarke — DISMISSED** [review]
Haven't done any testing but the code looks all good.

File: styles/theme/colors/light.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const newTheme = {
 			background: common.palette.neutral.n0,
 		},
 		secondary: {
-			background: common.palette.neutral.n10,
+			background: common.palette.alpha.white25,
```
[the remark this answers is not in the document]
Comment (koredefashokun) [reply styles/theme/colors/light.ts:9]: Yeah. I was confused myself. See attached screenshot:

image

[the remark this answers is not in the document]
Comment (avclarke) [reply styles/theme/colors/light.ts:9]: @asherism was this intentional on the design?

[the remark this answers is not in the document]
Comment (asherism) [reply styles/theme/colors/light.ts:9]: yes - it was intended as a way to lighten the odd cells against the n10 background of the table container. could be replaced with the n0 value if you'd prefer not to use alpha values.

[the remark this answers is not in the document]
Comment (koredefashokun) [reply styles/theme/colors/light.ts]: Thanks @asherism. I've pushed a commit that converts it to `n0` for convention purposes. @avclarke is there anything else keeping this from going in?

**Review by platschi — APPROVED** [review]

**Pull request merged**